### DISCUSSION TITLE
Unable to setup a free account

### DISCUSSION THREAD

**Discussion by rfennell** [testspace-com/support#19] [Q&A] [answered]
I have tried to set up a personal free account on my GitHub personal account. I was able to add Testspace via the marketplace but the call back to Testspace.com to complete the process failed

![image](https://user-images.githubusercontent.com/324718/130799884-668db6d3-3cdb-4e2b-90f0-aaf02ec09081.png)

Not sure what I should do now as it appears the testspace.com account has not been fully created. I seem to be able to sign in but only the samples are shown, there is no add new project.

What should I do to resolve this?

**Comment by munderseth** [comment, marked as answer]
Hi @rfennell I responded via email. Adding our email thread below.

---

From: mark@testspace.com <mark@testspace.com> 
Sent: Thursday, August 26, 2021 10:12 AM
To: 'Richard Fennell' 
Cc: mark@testspace.com
Subject: RE: Problem setting up an account

Hi Richard,

This is Mark from Testspace.  Sorry I missed you discussion yesterday. 

YES we introduced an account creation problem with our latest deployment. Thanks for pointing this out, was not aware of it. 

I can give you a workaround now. Most likely will not be officially fixed until next week.

I created an account for you outside of the Marketplace:  https://rfennell.testspace.com. 

-	Sign in using “Login with GitHub”
-	Next go to the Account tab
-	Select Services 
-	Select the Connect using your credentials – If you can select “Only select repositories” select it and then go back to “All repositories” and hit Save. Most likely this will not be available.
-	Go back to  https://rfennell.testspace.com
-	Again go to Account/Services and select your organization. Example below. 
 
![image](https://user-images.githubusercontent.com/1229367/131006877-fe5da2bd-1d26-427d-ab89-28e05a407f0c.png)

At this point things should work.

Sorry about this inconvenience. 

Note I am going to move this email thread to our discussion. Prefer to the discussion vs email.

Thanks,
>Mark

From: Richard Fennell <richard@blackmarble.co.uk> 
Sent: Thursday, August 26, 2021 8:52 AM
To: contact@testspace.com
Subject: Problem setting up an account

I wanted to have a look at your product as a potential solution to provide similar functionality Azure DevOps Test Plans for clients migrating to GitHub. So I have tried to link your system to my personal  GitHub to explore the features.

However, I have been having problems, the account setup fails (logged a discussion https://github.com/testspace-com/testspace.support/discussions/19)

The problem is 

have tried to set up a personal free account on my GitHub personal account. I was able to add Testspace via the marketplace but the call back to Testspace.com to complete the process failed
 
![image](https://user-images.githubusercontent.com/1229367/131006827-f08d3b7d-8336-48e3-a5e6-f0098c04389c.png)

Not sure what I should do now as it appears the testspace.com account has not been fully created. I seem to be able to sign in but only the samples are shown, there is no add new project.
What should I do to resolve this?

**Reply by rfennell** [reply]
@munderseth thanks for providing the workaround. I have the account setup now and linked to GitHub

**Reply by munderseth** [reply]
This has been resolved by GitHub. Refer info - https://github.community/t/webhooks-are-not-triggered-on-purchase-action/198456.
